docs/tutorials/stateless-application/hello-minikube.md

```diff
@@ -291,6 +291,12 @@ Build a new version of your image:
 docker build -t hello-node:v2 .
 ```
 
+Remove your previous Deployment:
```

Comment:
I'd prefer adding this after L310. @PascalBourdier WDYT?

Reply:
I think the "deployment" of v1 have to be deleted before the next deployment (v2) and avoid error : 
Error from server (AlreadyExists): deployments.extensions "hello-node" already exists

An another possibility is a link to the "Clean up" chapter ?

Reply:
@janetkuo WDYT?

Reply:
@PascalBourdier the series in which you're deleting the node would fail with this command in play - you're deleting the deployment, and then immediately trying to update it's `image` setting in the next command. 

The content for removing after you're all done is in the clean-up section already, so I don't think this is adding any value